standard/lexical-structure.md

```diff
@@ -755,7 +755,7 @@ Operator_Or_Punctuator
     | '+'  | '-'  | '*'  | '/'  | '%'   | '&'  | '|'  | '^'  | '!'  | '~'
     | '='  | '<'  | '>'  | '?'  | '??'  | '::' | '++' | '--' | '&&' | '||'
     | '->' | '==' | '!=' | '<=' | '>='  | '+=' | '-=' | '*=' | '/=' | '%='
-    | '&=' | '|=' | '^=' | '<<' | '<<=' | '=>'
+    | '&=' | '|=' | '^=' | '<<' | '<<=' | '=>' | '?.' | '?[' | '?('
```

Comment:
If my suspicions are correct, we'll be able to remove `?(`.

Reply:
Agree